docs/ref/forms/fields.txt

```diff
@@ -369,6 +369,13 @@ For each field, we describe the default widget used if you don't specify
     If provided, these arguments ensure that the string is at most or at least
     the given length.
 
+.. versionadded:: 1.8
```

Comment:
Not sure this can still go into 1.8,  ask Tim but you probably need to update to 1.9.